auth-server-open-api-spec.yaml

```diff
@@ -830,6 +832,33 @@ components:
           type: string
           description: Incoming payments MUST be created using this external reference.
       description: 'Open Payments specific property that defines the limits under which incoming payments can be created. '
+    access-quote:
+      title: access-quote
+      type: object
+      properties:
+        type:
+          type: string
+          description: The type of resource request as a string.  This field defines which other fields are allowed in the request object.
+          enum:
+            - quote
+        actions:
+          type: array
+          description: The types of actions the client will take at the RS as an array of strings.
+          items:
+            type: string
+            enum:
+              - read
+              - create
```

Comment:
Do we want to allow listing quotes?

Reply:
What would be the use case?

Reply:
The OP spec has an [endpoint for listing quotes](https://docs.openpayments.guide/reference/list-quotes), I'm curious what the use case for that was initially.

Reply:
That may have been a copy-paste error

Reply:
I didn't think there was one, but I deferred to the spec after seeing this. It seems like we should remove it(?)

Reply:
Let's remove it.